ci/ext/pre_env.d/kabanero_pre_env.sh

```diff
@@ -28,3 +28,12 @@ fi
 if [ -z "${LATEST_RELEASE}" ]; then
     export LATEST_RELEASE=true
 fi
+if [ "$TRAVIS" == "true" ]
+then
+    if [ $TRAVIS_TAG ] && [[ $TRAVIS_TAG =~ ^.*-(alpha|beta|rc)\.[0-9]* ]]
+    then
+        export IMAGE_REGISTRY_USERNAME="${IMAGE_REGISTRY_USERNAME}"beta
```

Comment:
This leaves me a little uncomfortable as we a prescribing that a user has to have a username with `beta` on the end of it to access their pre-release registry.

How would this work if the registry required authentication using a valid email address?

Could we use a different environment variable rather than overloading the default username and then set `IMAGE_REGISTRY_USERNAME="$BETA_REGISTRY_USERNAME"`?

Reply:
OK. What should we do if BETA_REGISTRY_USERNAME is not setup, error or continue using existing  value used in IMAGE_REGISTRY_USERNAME?

Reply:
Was looking at just doing something like this:
```
if [ "$TRAVIS" == "true" ]
then
    if [ $TRAVIS_TAG ] && [[ $TRAVIS_TAG =~ ^.*-(alpha|beta|rc)\.[0-9]* ]]
    then
        if [ -n "${BETA_REGISTRY_USERNAME}" ]; then
            export IMAGE_REGISTRY_USERNAME="${BETA_REGISTRY_USERNAME}"
        fi
        if [ -n "${BETA_REGISTRY_ORG}" ]; then
            export IMAGE_REGISTRY_ORG="${BETA_REGISTRY_ORG}"
        fi
    fi
fi
```

Reply:
I also presume the password for the beta docker registry may be different so we would need to have a  different variable for that also, ie BETA_REGISTRY_PASSWORD

Reply:
I think having the separate vars for the org, user and password is a good idea. If we are running a beta build and they are not set then we should log an error and terminate the build.

Reply:
Updated